service/lib/dinstaller/dbus/storage/manager.rb

```diff
@@ -108,6 +111,13 @@ def volume_templates
           proposal.volume_templates.map { |v| converter.to_dbus(v) }
         end
 
+        # Path of the D-Bus object containing the calculated proposal
+        #
+        # @return [::DBus::ObjectPath] Proposal object path or root path if no exported proposal yet
+        def result
+          dbus_proposal&.path || ::DBus::ObjectPath.new("/")
```

Comment:
It is a pitty that D-Bus does not have a better way to do this because returning the `/` looks like a workaround to me.

Reply:
Yes, maybe in the future we realize that the `#Result` property is not actually needed.